### ISSUE TITLE
Issue with Audio Slideshow audio player display when navigating back to the page (develop branch)

### ISSUE THREAD

**Issue by ronm123** [thexerteproject/xerteonlinetoolkits#640]
Hi all
I can't spot where this is being caused but it seems similar to an issue we had before when navigating back to pages and the display being corrupted. Here's an example and how to replicate...

Add an audio slideshow to a page with at least one page either side of that page
Navigate to the audio slideshow page and the display of the audio player is correct e.g.
![image](https://user-images.githubusercontent.com/5167146/27533620-235747ee-5a5c-11e7-9d48-04044f9028ec.png)
Navigate past that page to the next page and then back to the audio slideshow page. The display of the audio player is now messed up e.g.
![image](https://user-images.githubusercontent.com/5167146/27533667-4c286392-5a5c-11e7-865c-6bc0a0abbefb.png)
Here's a simple test with the audio slideshow on page 2
https://training.mitchellmedia.co.uk/xot/play.php?template_id=297#resume=1 
We noticed this in preparation for our next FT Talk on Wednesday so hoping we can find the cause/solution before then.

**Comment by ronm123** [comment]
@torinfo any thoughts on this one? Are you able to replicate? As far as my testing goes this isn't server or installation specific I can replicate it consistently including on xampp.

**Comment by torinfo** [comment]
@ronm123 Sorry, no, no ideas. Also, not much time this week.

**Comment by FayCross** [comment]
@ronm123 I'll have a look at this but if you need a very quick (half) fix before your talk today then comment out the loadAudio line in the pageChanged function. This means it will show fine on returning to the page as long as you haven't changed the size of the LO before going back to the page (so most of the time it should work ok). I'll look at a proper fix when i have some more time this afternoon.

**Comment by ronm123** [comment]
@FayCross thanks that is slightly better especially as we have a sequence of 3 audio slideshows where viewers might navigate back and forth between them.

**Comment by FayCross** [comment]
I'm a bit worried that there seem to suddenly be quite a few page types where they show fine initially but then go odd on later views e.g. matching texts messes up on 2nd views & accordion navigator messes up on 2nd view if the project size has changed. These all worked fine on previous versions.

Have we changed the order that things are being done in xenith at all? I'm wondering if it's to do with the timing of functions not being quite right anymore. I'll try to look into it - could it be anything to do with the separating of css @torinfo?

**Comment by FayCross** [comment]
@ronm123 I've committed a proper fix for this that involves a change to xenith rather than the page model file so you'll have to undo the change I recommended yesterday. It was caused by pageDiv being hidden initially when the page reloads and therefore the model files weren't receiving the correct dimensions of elements needed for resizing etc. I think the hide was put into stop an issue where pages were visibly changing when model css files were added / removed but I've just changed the timing of it and it seems ok I think.

It's also fixed the problems I found on the other pages
